Enterprise-CMCS/macpro-mdct-hcbs
# Favicon - Pt 2

### Description

In order for Github (or whatever the magic behind the scenes is called) to recognize the new assets for favicon, the first [PR](https://github.com/Enterprise-CMCS/macpro-mdct-hcbs/pull/52) was merged in with a different name - because if the name was the same, Github detected no changes on the branch. 

- This PR reverts the name back to default. 
- Favicon was updated via this [favicon generator website](https://favicon.io/favicon-converter/). It's super neat. You just upload the image and it returns a zip folder with several favicon images in different sizes, but for ours - we are only using favicon-16xx16.png, favicon 32x32.png and favicon.ico
- <img width="250" alt="folder with different favicon sizes" src="https://github.com/user-attachments/assets/4dd27622-9b83-4735-b889-144d56e105cf">

### Related ticket(s)


CMDCT-4092

---
### How to test

**Deploy Link:** https://d3eb8p5pv0lhv0.cloudfront.net/
1. As soon as you load the app even before you login, you should notice the favicon updated. 
2. It's super tiny, but you should still be able to make out "HCBS" from the logo
3. Favicon will show up in tab - no changes

| No change |
| ----- |
| <img width="300" alt="favicon for HCBS logo" src="https://github.com/user-attachments/assets/3525f558-18b6-4bdf-b54e-3e39488a6ca5"> |

---
### Author checklist
